src/components/NotAuthorized.tsx

```diff
@@ -0,0 +1,92 @@
+import { Split, SplitItem, StackItem } from '@patternfly/react-core';
+import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { NotAuthorized } from '@redhat-cloud-services/frontend-components';
+import { getInsights, localUrl } from '@redhat-cloud-services/insights-common-typescript';
+import { default as React, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { useLocation, useParams } from 'react-router-dom';
+
+import Config from '../config/Config';
+import messages from '../properties/DefinedMessages';
+import { useGetBundles } from '../services/Notifications/GetBundles';
+import { Facet } from '../types/Notification';
+import { getSubApp } from '../utils/Basename';
+
+interface NotificationListPageParams {
+    bundleName: string;
+}
+
+export const NotAuthorizedPage: React.FunctionComponent = () => {
+
+    const { bundleName } = useParams<NotificationListPageParams>();
+    const getBundles = useGetBundles();
+    const bundles: Facet | undefined = useMemo (() => {
+        if (getBundles.payload?.status === 200) {
+            return getBundles.payload.value.find(b => b.name === bundleName);
+        }
+    }, [ bundleName, getBundles.payload?.status, getBundles.payload?.value ]);
+
+    const location = useLocation();
+    const intl = useIntl();
+
+    const userPreferences = <a href={ localUrl(`/user-preferences/notifications/`,
+        getInsights().chrome.isBeta()) }> User Preferences</a>;
+    const myUserAccess = <a href={ localUrl(`/settings/my-user-access?bundle=rhel }`,
+        getInsights().chrome.isBeta()) }> My User Access </a>;
+
+    const serviceName = React.useMemo(() => {
+        switch (getSubApp(location.pathname)) {
+            case Config.integrations.subAppId:
+                return intl.formatMessage(messages.integrations);
+            case Config.notifications.subAppId:
+                return intl.formatMessage(messages.notifications);
+            default:
+                return '';
+        }
+    }, [ intl, location.pathname ]);
+
+    const pageHeaderTitleProps = {
+        paddingBottom: '8px'
+    };
+
+    const title = React.useMemo(() => {
+        if (serviceName === 'Notifications') {
+            return `${ serviceName } | ${ bundles?.displayName }`;
```

Comment:
`bundles?.displayName` will be undefined when it's loading. use a <Skeleton> in that case.

Reply:
use a what?

Reply:
sorry - a Skeleton